tests/test.ts

```diff
@@ -223,6 +223,47 @@ test("init and account fetch", async () => {
   }
 })
 
+
```

Comment:
You have an extra newline here which fails the linter. I should probably create a lint check script.